src/test/java/org/example/AthleteTest.java

```diff
@@ -0,0 +1,98 @@
+package org.example;
+
+import junit.framework.TestCase;
+
+public class AthleteTest extends TestCase {
+
+    public void testGetBonusTime() {
```

Comment:
name methods like this givenSomeConditionsExpectingSomeResult. so.. givenValidAthleteExpectingCorrectPenaltyScore